modules/adriverIdSystem.js

```diff
@@ -73,7 +73,17 @@ export const adriverIdSubmodule = {
             callback();
           }
         };
-        ajax(url, callbacks, undefined, {method: 'GET'});
+        let id5obj = JSON.parse(decodeURIComponent(storage.getDataFromLocalStorage('id5id')));
+        let universalVal;
+        for (let key in id5obj) {
+          if (id5obj.hasOwnProperty(key) && key === 'universal_uid') {
+            universalVal = id5obj[key];
+          }
+        }
+        let sharedID = storage.getDataFromLocalStorage('_pubcid') || storage.getCookie('pubcid');
+        let unifiedID = storage.getCookie('__uid2_advertising_token');
+        let newUrl = url + '&custom=' + (universalVal ? '221=' + encodeURIComponent(universalVal) + ';' : '') + (sharedID ? '222=' + encodeURIComponent(sharedID) + ';' : '') + (unifiedID ? '223=' + encodeURIComponent(unifiedID) : '') + '&cid=' + (storage.getDataFromLocalStorage('adrcid') || storage.getCookie('adrcid'));
```

Comment:
you can't do this

Reply:
please see id module rules at https://docs.prebid.org/dev-docs/module-rules.html